lib/ansible/modules/network/eric_eccli/eric_eccli_command.py

```diff
@@ -0,0 +1,246 @@
+      - Configures the interval in seconds to wait between retries
+        of the command. If the command does not pass the specified
+        conditions, the interval indicates how long to wait before
+        trying the command again.
+    default: 1
+"""
+
+EXAMPLES = r"""
+tasks:
+  - name: run show version on remote devices
+    eric_eccli_command:
+      commands: show version
+
+  - name: run show version and check to see if output contains IPOS
+    eric_eccli_command:
+      commands: show version
+      wait_for: result[0] contains IPOS
+
+  - name: run multiple commands on remote nodes
+    eric_eccli_command:
+      commands:
+        - show version
+        - show running-config interfaces
+
+  - name: run multiple commands and evaluate the output
+    eric_eccli_command:
+      commands:
+        - show version
+        - show running-config interfaces
+      wait_for:
+        - result[0] contains IPOS
+        - result[1] contains management
+
+  - name: run commands that require answering a prompt
+    eric_eccli_command:
+      commands:
+        - command: 'config'
+        - command: 'system hostname ub4-1-changed'
+        - command: 'commit'
+          prompt: 'Uncommitted changes found, commit them? [yes/no/CANCEL]'
+          answer: 'no'
+        - command: 'end'
+
+  - name: Set the prompt and error information regular expressions
+    eric_eccli_command:
+      commands:
+        - command: 'evr_2d01_vfrwd-evr1#dd'
+          prompt: 'error input: element does not exist'
+        - ansible.cfg:
+        - command: '[\r\n]+ error input: .*'
+
+        - command: 'evr_2d01_vfrwd-evr1#aaa'
+          prompt: 'aaa#'
+        - ansible.cfg:
+        - command: 'a{3}?#'
+"""
+
+RETURN = """
+stdout:
+  description: The set of responses from the commands
+  returned: always apart from low level errors (such as action plugin)
+  type: list
+  sample: ['...', '...']
+stdout_lines:
+  description: The value of stdout split into a list
+  returned: always apart from low level errors (such as action plugin)
+  type: list
+  sample: [['...', '...'], ['...'], ['...']]
+failed_conditions:
+  description: The list of conditionals that have failed
+  returned: failed
+  type: list
+  sample: ['...', '...']
+"""
+import re
+import time
+
+from ansible.module_utils._text import to_text
+from ansible.module_utils.network.eric_eccli.eric_eccli import run_commands
+from ansible.module_utils.network.eric_eccli.eric_eccli import eric_eccli_argument_spec, check_args
+from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils.network.common.utils import ComplexList
+from ansible.module_utils.network.common.parsing import Conditional
+from ansible.module_utils.six import string_types
+
+
+def to_lines(stdout):
+    for item in stdout:
+        if isinstance(item, string_types):
+            item = str(item).split('\n')
+        yield item
+
+
+def parse_commands(module, warnings):
+    command = ComplexList(dict(
```

Comment:
You should probably use `transform_commands` from `ansible.module_utils.network.common.utils` instead.